web_responsive/readme/DESCRIPTION.rst

```diff
@@ -66,29 +41,11 @@ This module adds responsiveness to web backend.
 
   .. image:: ../static/img/chatter.gif
 
-* Scrollable dropdowns
-
-  .. image:: ../static/img/dropdown_scroll.gif
-
-* Kanban interface adopted to mobile
-
-  .. image:: ../static/img/kanban.gif
-
-* Calendar interface adopted to mobile
-
-  .. image:: ../static/img/calendar.gif
-
-* Interface is adapted dynamically on device rotation
-
-  .. image:: ../static/img/device_rotation.gif
-
 * Big inputs on form in edit mode
 
 **Features for desktop computers**:
 
 * Keyboard shortcuts for easier navigation,
-  **using `Alt + Shift + [NUM]`** combination instead of
-  just `Alt + [NUM]` to avoid conflict with Firefox Tab switching.
```

Comment:
Why has this been deleted?

Reply:
> bvanjeelharia Thanks for your effort I saw two minor css issues. Fields with form view effects have unnecessary space There is a problem with the option field of the tree view ![Adres-Screenshot 2023-02-13 at 12-36-12 Odoo - Eren Tarim Ergun Bey](https://user-images.githubusercontent.com/35401092/218500308-19fd4fa4-b2df-4fec-be9f-470a6c82509b.png) ![Tree-Screenshot 2023-02-13 at 12-36-58 Odoo - Contacts](https://user-images.githubusercontent.com/35401092/218500442-ef650041-e3ff-4d3f-9be6-af6a3498fa24.png)

@tate11 ,I have made changes for the extra width in formview fields and also the optional fields would now show up fine

Reply:
That was a mistake from my end , I have added it again